mantidimaging/core/reconstruct/cil_recon.py

```diff
@@ -69,6 +69,8 @@ def single_sino(sino: np.ndarray,
         Should return a numpy array,
         """
 
+        print(f"SPDHG params: {recon_params.stochastic=} {recon_params.subsets=}")
```

Comment:
It's a pretty minor thing but I wonder if it should show the subsets value when stochastic is false?

Reply:
This is just here so the CIL devs can see how to get the parameters. It will be removed from the final version.